airflow/cli/cli_parser.py

```diff
@@ -224,6 +224,10 @@ def _check(value):
     default=ColorMode.AUTO,
 )
 
+# DB args
+ARG_REVISION = Arg(("-r", "--revision"), default='heads', help="Revision target or range for --sql mode")
```

Comment:
```suggestion
ARG_REVISION = Arg(("-r", "--revision"), default='heads', help="Revision target or range (start:end) for --sql mode; ignored if --sql is not passed")
```

I think? Based on my understanding to the implementation.

Reply:
The `start:end` is correct. I made it so that if range is given in revision then automatically it's --sql mode. This is because range cannot be used in online mode